docs/modules/ROOT/pages/functions.adoc

```diff
@@ -0,0 +1,98 @@
+= Functions
+
+Cypher Builder implements the most common link:https://neo4j.com/docs/cypher-manual/current/functions/[Cypher Functions], such as `coalesce` or `count`. In these cases, you can directly call the provided functions to generate the equivalent Cypher:
+
+```javascript
+const movie = new Cypher.Node({labels: ["Movie"]});
+
+Cypher.coalesce(movie.property("title"), new Cypher.Param("The Matrix"));
+```
+
+```cypher
+coalesce(this0.title, $param0)
+```
+
+All the available functions will also have the correct signature for TypeScript.
+
+[NOTE]
+====
+By convention, all Cypher functions are available as JavaScript functions, with camelCase naming. This differentiates functions from clauses and other elements of the Cypher Builder.
+For instance, the function link:https://neo4j.com/docs/cypher-manual/current/functions/aggregating/#functions-count[`count()`] and the link:https://neo4j.com/docs/cypher-manual/current/syntax/expressions/#count-subqueries[`new Count()`] subquery.
+====
+
+Exposed functions will have the closest possible name to their Cypher counterpart, and reside in the same namespace, for example:
+
+```javascript
+Cypher.apoc.cypher.runFirstColumnSingle()
+```
+
+Is equivalent to the Cypher function:
+
+```cypher
+apoc.cypher.runFirstColumnSingle()
+```
+
+== Custom Functions
+
+In some cases, you may need to use functions that are not available in Cypher Builder, for instance, if you are using plugins or some of the newest features of Neo4j.
+
+For these cases, you can use the class `Function` to create custom functions. As an example, let's call the function link:https://neo4j.com/docs/cypher-manual/current/functions/mathematical-numeric/#functions-isnan[`isNan`] as a custom function.
+
+
+```javascript
+const isNan = new Cypher.Function("isNan", [new Cypher.Literal(0)]);
+```
+
+```cypher
+isNan(0);
+```
+
+The class `Function` takes 2 parameters:
+
+* **name**: The name of the function.
```

Comment:
Must these parameters be in lower case? Are they written the way it's shown in the code? If so, then maybe we should format it like code, between `

Reply:
Good catch, I'll change the formatting to use `code`